### PR TITLE
added new query for user_missing_group

### PR DESCRIPTION
**Pull request by soniantao**
~Closes 828~

### CODE REVIEW

**Comment by rogeriopeixotocx** [comment]
Closes #827

File: assets/queries/cloudFormation/user_missing_group/metadata.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "user_missing_group",
+  "queryName": "user_missing_group",
```
Comment (ruigomescx) [inline assets/queries/cloudFormation/user_missing_group/metadata.json:3]: Query name shouldn't have underscores. A more suitable name: "IAM User With No Group"

File: assets/queries/cloudFormation/user_missing_group/metadata.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  "severity": "LOW",
+  "category": null,
+  "descriptionText": "A user should belong to a group",
```
Comment (ruigomescx) [inline assets/queries/cloudFormation/user_missing_group/metadata.json:6]: A **IAM** user

File: assets/queries/cloudFormation/user_missing_group/query.rego [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+             }
+}
+CxPolicy [result] {
+  resource := input.document[i].Resources[name]
+  resource.Type == "AWS::IAM::User"
+  groups := resource.Properties.Groups
+  groups != []
+  result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("Resources.%s.Properties", [name]),
```
Comment (ruigomescx) [inline assets/queries/cloudFormation/user_missing_group/query.rego:23]: We can point here to the GroupsKey, to have a more accurate result:
Resources.%s.Properties.**Groups**

**Review by ruigomescx — CHANGES_REQUESTED** [review]

File: assets/queries/cloudFormation/user_missing_group/query.rego (hunk 3 above)
Comment (soniantao) [reply assets/queries/cloudFormation/user_missing_group/query.rego]: fixed

**Review by ruigomescx — APPROVED** [review]

**Pull request merged**